File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/InvalidAuthorizationHeaderException.java
package com.dogGetDrunk.meetjyou.common.exception.business;

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode;

public class InvalidAuthorizationHeaderException extends CustomJwtException {

    private String value;

    public InvalidAuthorizationHeaderException(String value) {
        this(value, ErrorCode.INVALID_AUTHORIZATION_HEADER);
    }

    public InvalidAuthorizationHeaderException(String value, ErrorCode errorCode) {
        super(value, errorCode);
        this.value = value;
    }
}

File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/InvalidAuthorizationHeaderException.kt
package com.dogGetDrunk.meetjyou.common.exception.business

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode

class InvalidAuthorizationHeaderException(
    value: String,
    errorCode: ErrorCode = ErrorCode.INVALID_AUTHORIZATION_HEADER
) : CustomJwtException(value, errorCode)
